github-page/_docs/decorators/prop.md

```diff
@@ -541,20 +555,20 @@ class Something {
 
 // This would be mapped to
 {
-  type: [[[{ type: String }]]]
+  type: [[[{ type: String }]]];
 }
 ```
 
 ### Map Options
 
 #### of
 
-Accepts Type: `any`
-(alias for [`type`](#type) from `@prop`)
+Accepts Type: `any` (alias for [`type`](#type) from `@prop`)
```

Comment:
why did this get on one line?

Reply:
No idea. I have to also admit, I did this early morning on one cup of coffee. 😊 

Scott